baseservice/utils.py

```diff
@@ -92,38 +93,49 @@ def __init__():
     a.z # raises AttributeError (z was not set on class). once it's set, then value is the init value for each thread
     a.w # 2 for this thread, 1 for any other thread (because of the default init value)
     """
+    __NO_INIT_VALUE = None
 
-    def __init__(self, init_value: Optional[TValueType] = ...):
+    def __init__(self, init_value: Optional[TValueType] = __NO_INIT_VALUE):
```

Comment:
what's wrong with the ellipsis?

Reply:
Not that it's important anymore but usually sentinels are represented with object() instead of elipssis (which i think only pydantic does)